modules/ROOT/pages/dataweave-maven-plugin.adoc

```diff
@@ -0,0 +1,329 @@
+= Packaging and Deploying DataWeave Libraries
+
+Use the DataWeave Maven plugin to integrate the packaging and deployment of your DataWeave libraries with your Maven lifecycle.
```

Comment:
Apache can get persnickety about the use of their names: can we call this just the Maven plugin or the Apache Maven plugin for DataWeave?
```suggestion
Use the Apache Maven plugin for DataWeave to integrate the packaging and deployment of your DataWeave libraries with your Maven lifecycle.
```